<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiTranslatorXYZ</class>
 <widget class="QWidget" name="MultiTranslatorXYZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <layout class="QVBoxLayout" name="lyt_coorhub"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Translate the coordinates by (x, y, z):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spin_x_um">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spin_y_um">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spin_z_um">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Âµm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="chk_deleteori">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete original ROIs</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_commit">
         <property name="text">
          <string>Translate the selected ROIs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_instruction">
       <property name="text">
        <string>Show instructions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
